### PR TITLE
Page layouts refactoring

### PR DESCRIPTION
**Pull request by melniiv**
## Description
Refactoring/redesigning according to abstract frontpage, eventss search page, event and article pages.

### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

**Review by nenonja — APPROVED** [review]
Several minor comments, but nothing special so I approved as well. I see that it's quire important to test every breakpoint too, I hope it's tested well before merging. Or maybe it can be merged anyway as it's not in production.

File: src/domain/event/eventCard/LargeEventCard.tsx [hunk 1]
```diff
@@ -140,33 +156,48 @@ const LargeEventCard: React.FC<Props> = ({ event }) => {
...
+                    iconRight={<IconLinkExternal aria-hidden />}
+                    fullWidth
+                    onClick={goToBuyTicketsPage}
+                    size="small"
+                    variant="success"
+                  >
+                    {t('event:eventCard.buttonBuyTickets')}
+                  </Button>
+                </div>
+                <div ref={button}>
```
[the remark this answers is not in the document]
Comment (melniiv) [reply src/domain/event/eventCard/LargeEventCard.tsx:179]: yes, that was a requirement for the new design. in other cases only arrow button icon is shown

File: src/domain/search/landingPage/landingPage.module.scss [hunk 2]
```diff
@@ -5,56 +5,33 @@
 $mainBackground: var(--color-fog-medium-light);
 
 .layout {
-  --spacing: #{$spacing-layout-2-xs};
```
[the remark this answers is not in the document]
Comment (melniiv) [reply src/domain/search/landingPage/landingPage.module.scss:8]: seems that it was more about grid layout..

File: src/domain/search/eventSearch/filterSummary/__tests__/__snapshots__/DateFilter.test.tsx.snap [hunk 3]
```diff
@@ -31,14 +32,14 @@ exports[`matches snapshot 1`] = `
...
   </button>
-  text
 </div>
 `;
 
 exports[`matches snapshot 2`] = `
 <div
   class="filter date"
 >
+  dateSelector.dateTypeValue
```
[the remark this answers is not in the document]
Comment (melniiv) [reply src/domain/search/eventSearch/filterSummary/__tests__/__snapshots__/DateFilter.test.tsx.snap:42]: fixed

**Pull request merged**